Documentation/Signoffs/PowerDistributionSystem.md

```diff
@@ -0,0 +1,82 @@
+Team 4 – Mario Kart & Ride Replay Simulator
+
+Team Member: Ray Durlin
+
+Detail Design: Main Power Distribution System
+
+
+Function of the Subsystem: 
+
+The function of the main power distribution system is to supply protected power to the following components: main microcontroller, user interface microcontroller, microcontroller cooling fans, Nintendo Switch, resistance system linear motor, and lastly the display monitor. The input power is to be received from a wall receptable via a cannon plug connection at the rear of the user interface enclosure. The input voltage is to be 120 volts ac, 60 Hz, circuit protected. A circuit breaker, CBM, is to be installed after the 120 VAC input from the receptacle so that the system can be locked out when any enclosure covers are taken off as a safety precaution. After the input power from the receptable enters the user interface enclosure, the power will be directed to a three of AC to DC converters. Converters C1, C2, and C3 listed on the schematic will convert input voltage from 120 VAC to an output voltage 12 VDC, 5 VDC, and 15 VDC, respectively. The output power by the converters C1, C2, and C3 will then enter three individual mini-fuses, F1, F2, and F3 prior to supplying power to any devices downstream for protecting each device. 
+
+
+Constraints:
+
+Reliability: The power distribution must provide reliable power to all devices while maintaining the specified protection ratings set by the National Electric Code (NEC). The system will maintain overprotection of the simulator devices with a rating of %125 percent of continuous circuit load as specified by the National Electrical Code Article 210.20(A) [1]. The overprotection rating of each circuit breaker will ensure that all devices are properly protected without the risk of opening the circuit unintentionally as circuit load fluctuates. Unintentionally opening a circuit due to inaccurate load calculations results in decreased system reliability. 
+
+Safety: Safety is the most important aspect of this subsystem as there is the potential for electrocution if not designed properly. To mitigate the possibility of electrocution by anyone working inside the user interface. All 120 VAC terminals within the user interface are to have been covered and contain a warning of electrocution on the cover. It is recommended that lock out tag out procedures to be followed when working within the user interface enclosure with any covers off. All wire sizing is to be in accordance with National Electrical Code Article 240.4(B) based on continuous circuit load [1]. 
+
+Buildable schematic:
+
+![image](https://user-images.githubusercontent.com/114370750/203234571-c7df044e-180e-413c-b995-9f830cb3505d.png)
```

Comment:
Why is ground going through a fuse?

single pole circuit breakers are all that is needed for conductor protection in this scenario.

Reply:
Not sure why I installed a fuse on ground, I have corrected this issue. 

Smallest GFCI DIN rail i could locate was rated for 15 A. Decided to go with the GFCI in the cord as you previously mentioned at 15 A.